pkg/provider/tracing.go

```diff
@@ -23,11 +23,13 @@ func tr(resource func() *schema.Resource) *schema.Resource {
 		defer span.Finish()
 		return so.ReadContext(ctx, rd, m)
 	}
-	sc.UpdateContext = func(ctx context.Context, rd *schema.ResourceData, m interface{}) diag.Diagnostics {
-		span := sentry.StartSpan(ctx, "terraform.resource.update", sentry.WithTransactionName("terraform.resource"))
-		span.Description = "Resource update"
-		defer span.Finish()
-		return so.UpdateContext(ctx, rd, m)
+	if so.UpdateContext != nil {
+		sc.UpdateContext = func(ctx context.Context, rd *schema.ResourceData, m interface{}) diag.Diagnostics {
+			span := sentry.StartSpan(ctx, "terraform.resource.update", sentry.WithTransactionName("terraform.resource"))
+			span.Description = "Resource update"
+			defer span.Finish()
+			return so.UpdateContext(ctx, rd, m)
+		}
```

Comment:
This change is required because `authentik_outpost_provider_attachement` consists only from ForceNew fields, so terraform provider check prohibits having UpdateContext on this resource